action.yml

```diff
@@ -40,6 +40,32 @@ inputs:
   force-new-deployment:
     description: 'Whether to force a new deployment of the service. Valid value is "true". Will default to not force a new deployment.'
     required: false
+  #one off task inputs
+  run-task:
+    description: 'Whether to run the task outside of an ECS service. Task will run before the service is updated if both are provided. Will default to not run.'
```

Comment:
nit: Is `outside of an ECS service` imply that the task isn't run wihtin ECS at all? Alternatively, should this read something like: `Whether to run as a stand-alone task in a ECS cluster`?